source/manual/data-gov-uk-architecture.html.md

```diff
@@ -66,6 +68,19 @@ cf env publish-data-beta-production
 
 We use [GOV.UK Signon][signon] for user authentication in [Publish Data][publish], with the app in each environment linked to the corresponding instance of [GOV.UK Signon][signon]. See the Publish ADR for more info.
 
-## [CKAN]
+## [Legacy CKAN (ckanext-dgu)][legacy-ckan]
 
-TODO
+[Legacy CKAN][legacy-ckan] is an extension to CKAN 2.2b and is provisioned on Bytemark hosting.  The following servers are in use:
+
+* Staging: `co-prod2.dh.bytemark.co.uk`
+* Production: `co-prod3.dh.bytemark.co.uk`
+
+User authentication is handled within Drupal, which sits alongside the highly customised CKAN installation.
+
+Legacy CKAN will be retired when the CKAN upgrade is launched.
+
+## [CKAN (ckanext-datagovuk)][ckan]
+
+This is a plugin to CKAN 2.7.3, as an interim solution before [Publish] is launched. Drupal has been removed and user authentication is handled wholly within CKAN.
+
+TODO: Details about staging/production environments
```

Comment:
Is this going to be a separate PR?

Reply:
Yes, I'll update the docs with the staging/prod environments when they have been fully set up